pkg/sentry/platform/ring0/aarch64.go

```diff
@@ -94,8 +94,9 @@ const (
 
 // System call vectors.
 const (
-	Syscall   Vector = El0Sync_svc
-	PageFault Vector = El0Sync_da
+	Syscall                 Vector = El0Sync_svc
+	PageFault               Vector = El0Sync_da
+	VirtualizationException Vector = El0Error
```

Comment:
This doesn't build because VirtualizationException is already defined above on L91.

Reply:
Hi,
Thanks for your checking.
Done.